src/graphql/node/control.rs

```diff
@@ -869,23 +899,48 @@ mod tests {
         let res = schema
```

Comment:
This comment should be modified according to the change; No longer returning of gigantoDraft.

Reply:
Thank you so much 🙇 I updated the comment for `apply_node` function accordingly as well.